libsplinter/src/rest_api/events.rs

```diff
@@ -60,14 +61,14 @@ impl<T: Serialize + Debug + Clone + 'static> EventDealer<T> {
         Ok(Response::from(res))
     }
 
-    /// Send message to all created WebSockets
-    pub fn dispatch(&mut self, msg: T) {
+    /// Send event to all created WebSockets.
+    pub fn dispatch(&mut self, event: T) {
         self.senders.retain(|sender| {
-            if let Err(err) = sender.unbounded_send(MessageWrapper::Message(msg.clone())) {
+            if let Err(err) = sender.unbounded_send(MessageWrapper::Message(event.clone())) {
                 error!("Dropping sender due to error: {}", err);
                 false
             } else {
-                trace!("Message sent: {:?}", msg);
+                trace!("Message sent: {:?}", event);
```

Comment:
This log message will be duplicated for every sender, but without any information of who received - we should only log this once.

Reply:
Moved log statement to the end of the method